<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>461</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Вывести результат</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
